### DISCUSSION TITLE
Permenant black screen after sleep mode

### DISCUSSION THREAD

**Discussion by sebboop** [eduardofilo/RG350_adam_image#116] [General]
I put my 280V into sleep mode while I was using the pscx4all emulator. When I tried to wake my device it wouldn't wake up. Just shows black screen with glow. I reset it but now it boots to black screen (green light is on and screen is glowing).
I can access it through MTP via cable and files seems fine there but all my gxmenu icons have disappeared when I check screenshots.
Flashing new firmware or using different SD card also doesn't fix it unfortunately. 
I also get this issue now when updating with any firmware other than the clean orginal firmware (that works fine).
Hope someone can help!

**Comment by eduardofilo** [comment]
At first I would tell you that the reset corrupted the card, which is possible. But I don't understand that flashing again even on different cards produces the same problem. Have you tried to also remove the external card just in case it also experienced some kind of corruption and now it affects the system when accessing it?
If with new cards just flashed (INT) and formatted (EXT) there are still problems, something must have happened physically to your console. I can't think of anything else without having access to the machine.
Also try cleaning the contacts of the cards with isopropyl alcohol, which sometimes accumulates grease from the fingers that later ends up contaminating the internal contacts of the machine.

**Comment by leofunaki** [comment]
I am having the same issue with PSCX4all, my RG280V will not boot up after this black screen of death. Battery also gets really hot. When you say something must have happened physically to my console, what do you mean? Are there steps I can take to determine what the issue is?

**Comment by eduardofilo** [comment]
I said that because is the only explanation that occurs to me when the console doesn't work with a new fresh flashed SD card. But who knows. It is difficult to make a diagnosis without having access to the machine in these weird cases.

**Comment by mchenier** [comment]
I think I have the same behavior. I have 1.2 was playing PSX game it was in sleep mode. After that I think I was out of battery because I had a black screen when I tried to wake it up. I plugged it and after a couple hours I booted it up. I saw an interface with 2 random game and nothing else I can't do anything in that state. I assume it's corrupted. I then tried without the EXT sd card and I had the same problem. Now I'm trying to recover my save game of psx4all but it's not an easy task. Disk Genius seams to be to solution since it's the only program I can see the files in /local/home/.psx4all but I can't recover the file unless I buy the program for 70$US. Anyone has a solution for me ?

**Reply by eduardofilo** [reply]
Sleep mode consumes a lot more battery than it seems. It is only advisable to use it to pause the system for a few minutes. But if several hours are going to pass, it is better to turn off or there is a risk that what you say will happen.

**Comment by mchenier** [comment]
I found the solution. When the menu is bugged out you need to delete the /media/data/local/home/.simplemenu/last_state.sav .
To do that if you don't have access to the Command App you can use the free version of Disk Genius. You should see the file and delete it trough the app.
After that you should be able to boot to a menu a start the PyBackup application. Backup all the files you want they will be store in the /backups folder in the EXT sdcard. After that the simplest method is to reflash the INT sdcard and restore the backup afterward. I hope we find a fix for the corruption but at least I won't loose my savegame while I wait.

**Reply by eduardofilo** [reply]
In 1.3.1 you don't need to use DiskGenius to delete last_state.sav file. Pressing `B` during boot, GMenu2X will be opened instead of SimpleMenu. That way you will have access to Commander to delete the file.
DiskGenius is well known by corrupting Linux filesystems. It is better to use only for reading those filesystems, not to modify.
